### DISCUSSION TITLE
feature request - disable auto update check in settings?

### DISCUSSION THREAD

**Discussion by d00lar** [seerge/g-helper#883] [Ideas]
hello
i think it should be user choice in settings - do user wants that  if we open an app and there is update then it asks automaticly if i want download?

if im on good / working fine version for me - it is my decision if i want to check for update or not 

what do You think ? please consider
best regards !

**Comment by seerge** [comment]
What’s the practical purpose of that? If would click NO on update offer, app would remember that and don’t ask you again anyway

**Reply by d00lar** [reply]
Cause even  if we upgrade next relase will ask again. IT will also check in background even if not ask? I do not like if app do something in background that i do not wants. It is not my choice it s up to You:) app is great anyway but imho decent app like this one should have this option. Regads

**Reply by seerge** [reply]
@d00lar well, i can add an optional flag in config for that. It's just quite strange request in a first place :) as app would still do some things in background (like set performance mode or handle your hotkeys / etc) :)

**Reply by d00lar** [reply]
but not doing some http requests in background at least ;P flag in config is totaly fine for me if You can ;)  thank You

**Reply by seerge** [reply]
This build supports a new param in config

`` "skip_updates" : 1, ``

And you should be good to go

[GHelper.zip](https://github.com/seerge/g-helper/files/12188497/GHelper.zip)
